website-components-playground/src/components/knobs-controller.tsx

```diff
@@ -17,7 +17,7 @@ export type TKnob = {
   }[];
 };
 type TKnobsControllerFunctionOptions = {
-  form: JSX.Element;
+  form: React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.